src/main/resources/jupyter/init-actions.sh

```diff
@@ -106,6 +127,8 @@ if [[ "${ROLE}" == 'Master' ]]; then
     JUPYTER_USER_SCRIPT_OUTPUT_URI=$(jupyterUserScriptOutputUri)
     JUPYTER_NOTEBOOK_CONFIG_URI=$(jupyterNotebookConfigUri)
 
+    STEP_TIMINGS+=($(date +%s))
```

Comment:
how are these metrics being sent to newrelic?

Reply:
they are not. Since this is run on the cluster, we run into the same issue sending them to new relic as we do with welder metrics. You have to inspect them in the log on the cluster

Reply:
Seems like they're just logged, not sent to NR. We shouldn't put a NR key on users' clusters. This is a good first step; wonder if there is a solution for getting this data to NR.